### PR TITLE
Zab/quot proj indx

### PR DESCRIPTION
**Pull request by zabbo**

### CODE REVIEW

**Comment by zabbo** [comment]
Just making a PR out of this, while it's still a draft, to get some CI coverage

**Comment by versity-github** [comment]
Build 97 http://jenkins.vpn.versity.com:8080/job/scoutfs-run-pr-el8/97/ git: c0b8a63d38204cb0fda9a1554debc40012da2ca3

**Comment by versity-github** [comment]
Build 122 http://jenkins.vpn.versity.com:8080/job/scoutfs-run-pr/122/ git: c0b8a63d38204cb0fda9a1554debc40012da2ca3

File: tests/tests/quota.sh [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+}
+
+reset_all()
+{
+	rm -f "$FILE"
+	scoutfs quota-wipe -p "$T_M0"
+	getfattr --absolute-names -d -m - "$T_D0" | \
+		grep "^scoutfs.totl." | \
+		cut -d '=' -f 1 | \
+		xargs -n 1 -I'{}'  setfattr -x '{}' "$T_D0"
```
Comment (aversecat) [inline tests/tests/quota.sh:25]: on `el9` this croaks about `-n` and `-I` being mutually exclusive. I need to check whether my suspicion is right but I think that likely it has always been the case that `-I` implies `-n 1` anyway, even in old versions. So the fix will likely be that we just remove `-n 1` here.

Comment (aversecat) [reply tests/tests/quota.sh:25]: ok, just confirmed you can just remove `-n 1` and it'll work on `el7` as well as `el8` without issues as well.

**Comment by zabbo** [comment]
Nice, thanks for pointing these out.  I'll address this soon when the quota work gets moving again.

**Comment by versity-github** [comment]
Build 141 http://jenkins.vpn.versity.com:8080/job/scoutfs-run-pr/141/ git: c0b8a63d38204cb0fda9a1554debc40012da2ca3

**Comment by versity-github** [comment]
Build 116 http://jenkins.vpn.versity.com:8080/job/scoutfs-run-pr-el8/116/ git: d9637cc76f1679fdb42b1dfd766473ed6f2b3fa2

**Comment by versity-github** [comment]
Build 142 http://jenkins.vpn.versity.com:8080/job/scoutfs-run-pr/142/ git: d9637cc76f1679fdb42b1dfd766473ed6f2b3fa2

**Comment by versity-github** [comment]
Build 164 http://jenkins.vpn.versity.com:8080/job/scoutfs-run-pr/164/ git: 18f00a8acc1b3d1f4ce2f975d97f26e2948d0148

**Comment by versity-github** [comment]
Build 138 http://jenkins.vpn.versity.com:8080/job/scoutfs-run-pr-el8/138/ git: 4609365853cdd18514423b82372f2e9f7fdabb71

**Comment by versity-github** [comment]
Build 139 http://jenkins.vpn.versity.com:8080/job/scoutfs-run-pr-el8/139/ git: 18f00a8acc1b3d1f4ce2f975d97f26e2948d0148

**Comment by versity-github** [comment]
Build 171 http://jenkins.vpn.versity.com:8080/job/scoutfs-run-pr/171/ git: 18f00a8acc1b3d1f4ce2f975d97f26e2948d0148

**Comment by aversecat** [comment]
Obsoleted by #177

**Pull request closed without merging**